Jenkinsfile

```diff
@@ -0,0 +1,111 @@
+/**
+* IOS Jenkinsfile
+*/
+
+//     In RHMAP's case, the following parameters are sent by RHMAP to the Jenkins job.
+//     This means, the Jenkins job has to be a parametrized build with those parameters.
+CODE_SIGN_PROFILE_ID = params?.BUILD_CREDENTIAL_ID?.trim()                 // e.g. "redhat-dist-dp"
+BUILD_CONFIG = params?.BUILD_CONFIG?.trim()                                // e.g. "Debug" or "Release"
+
+//     To hardcode the values uncomment the lines below
+//CODE_SIGN_PROFILE_ID = "redhat-dist-dp"
+//BUILD_CONFIG = "Debug"
+
+// sample values commented below are for https://github.com/feedhenry-templates/helloworld-ios
+/* ------------- use these to hardcode values in Jenkinsfile ---------------- */
+PROJECT_NAME = "helloworld-ios-app"
+INFO_PLIST = "helloworld-ios-app/helloworld-ios-app-Info.plist"
+VERSION = "1.0.0"
+SHORT_VERSION = "1.0"
+BUNDLE_ID = "com.feedhenry.helloworld-ios-app"
```

Comment:
I think those values need to be checked again. They are not matching this project.

Reply:
lets use `org.aerogear.ios-showcase-template`